### PR TITLE
Add websphere-liberty 18.0.0.4 images

### PR DESCRIPTION
**Pull request by naumanna**
Add 18.0.0.4 websphere-liberty images, and update non-version tags to point to 18.0.0.4.

### CODE REVIEW

File: library/websphere-liberty [hunk 1]
```diff
@@ -2,35 +2,62 @@ Maintainers: Wendy Raschke <wraschke@us.ibm.com> (@wraschke),
...
+Tags: 18.0.0.4-springBoot1, springBoot1
+Directory: ga/18.0.0.4/springBoot1
+
+Tags: 18.0.0.4-webProfile7, webProfile7
+Directory: ga/18.0.0.4/webProfile7
+
+Tags: 18.0.0.4-javaee7, javaee7
+Directory: ga/18.0.0.4/javaee7
+
+Tags: 18.0.0.3-javaee8
```
Comment (tianon) [inline library/websphere-liberty]: Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (arthurdm) [reply library/websphere-liberty]: hi @tianon - we would like to keep the N-2 versions updated going forward, as customers need a bit of time to move up the versions, so we wanted those versions to stay current with the base changes, etc.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 46e103d..cfe8499 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,3 +1,39 @@
+websphere-liberty:18.0.0.3-javaee7 @ amd64
+websphere-liberty:18.0.0.3-javaee7 @ i386
+websphere-liberty:18.0.0.3-javaee7 @ ppc64le
+websphere-liberty:18.0.0.3-javaee7 @ s390x
+websphere-liberty:18.0.0.3-javaee8 @ amd64
+websphere-liberty:18.0.0.3-javaee8 @ i386
+websphere-liberty:18.0.0.3-javaee8 @ ppc64le
+websphere-liberty:18.0.0.3-javaee8 @ s390x
+websphere-liberty:18.0.0.3-kernel @ amd64
+websphere-liberty:18.0.0.3-kernel @ i386
+websphere-liberty:18.0.0.3-kernel @ ppc64le
+websphere-liberty:18.0.0.3-kernel @ s390x
+websphere-liberty:18.0.0.3-microProfile1 @ amd64
+websphere-liberty:18.0.0.3-microProfile1 @ i386
+websphere-liberty:18.0.0.3-microProfile1 @ ppc64le
+websphere-liberty:18.0.0.3-microProfile1 @ s390x
+websphere-liberty:18.0.0.3-microProfile2 @ amd64
+websphere-liberty:18.0.0.3-microProfile2 @ i386
+websphere-liberty:18.0.0.3-microProfile2 @ ppc64le
+websphere-liberty:18.0.0.3-microProfile2 @ s390x
+websphere-liberty:18.0.0.3-springBoot1 @ amd64
+websphere-liberty:18.0.0.3-springBoot1 @ i386
+websphere-liberty:18.0.0.3-springBoot1 @ ppc64le
+websphere-liberty:18.0.0.3-springBoot1 @ s390x
+websphere-liberty:18.0.0.3-springBoot2 @ amd64
+websphere-liberty:18.0.0.3-springBoot2 @ i386
+websphere-liberty:18.0.0.3-springBoot2 @ ppc64le
+websphere-liberty:18.0.0.3-springBoot2 @ s390x
+websphere-liberty:18.0.0.3-webProfile7 @ amd64
+websphere-liberty:18.0.0.3-webProfile7 @ i386
+websphere-liberty:18.0.0.3-webProfile7 @ ppc64le
+websphere-liberty:18.0.0.3-webProfile7 @ s390x
+websphere-liberty:18.0.0.3-webProfile8 @ amd64
+websphere-liberty:18.0.0.3-webProfile8 @ i386
+websphere-liberty:18.0.0.3-webProfile8 @ ppc64le
+websphere-liberty:18.0.0.3-webProfile8 @ s390x
 websphere-liberty:beta @ amd64
 websphere-liberty:beta @ i386
 websphere-liberty:beta @ ppc64le
@@ -18,10 +54,10 @@ websphere-liberty:microProfile1 @ amd64
 websphere-liberty:microProfile1 @ i386
 websphere-liberty:microProfile1 @ ppc64le
 websphere-liberty:microProfile1 @ s390x
-websphere-liberty:microProfile @ amd64
-websphere-liberty:microProfile @ i386
-websphere-liberty:microProfile @ ppc64le
-websphere-liberty:microProfile @ s390x
+websphere-liberty:microProfile2 @ amd64
+websphere-liberty:microProfile2 @ i386
+websphere-liberty:microProfile2 @ ppc64le
+websphere-liberty:microProfile2 @ s390x
 websphere-liberty:springBoot1 @ amd64
 websphere-liberty:springBoot1 @ i386
 websphere-liberty:springBoot1 @ ppc64le
diff --git a/_bashbrew-list b/_bashbrew-list
index af529d0..33f7119 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -7,12 +7,20 @@ websphere-liberty:18.0.0.3-springBoot1
 websphere-liberty:18.0.0.3-springBoot2
 websphere-liberty:18.0.0.3-webProfile7
 websphere-liberty:18.0.0.3-webProfile8
+websphere-liberty:18.0.0.4-javaee7
+websphere-liberty:18.0.0.4-javaee8
+websphere-liberty:18.0.0.4-kernel
+websphere-liberty:18.0.0.4-microProfile1
+websphere-liberty:18.0.0.4-microProfile2
+websphere-liberty:18.0.0.4-springBoot1
+websphere-liberty:18.0.0.4-springBoot2
+websphere-liberty:18.0.0.4-webProfile7
+websphere-liberty:18.0.0.4-webProfile8
 websphere-liberty:beta
 websphere-liberty:javaee7
 websphere-liberty:javaee8
 websphere-liberty:kernel
 websphere-liberty:latest
-websphere-liberty:microProfile
 websphere-liberty:microProfile1
 websphere-liberty:microProfile2
 websphere-liberty:springBoot1
diff --git a/websphere-liberty_javaee7/Dockerfile b/websphere-liberty_18.0.0.3-javaee7/Dockerfile
similarity index 100%
copy from websphere-liberty_javaee7/Dockerfile
copy to websphere-liberty_18.0.0.3-javaee7/Dockerfile
diff --git a/websphere-liberty_javaee7/server.xml b/websphere-liberty_18.0.0.3-javaee7/server.xml
similarity index 100%
copy from websphere-liberty_javaee7/server.xml
copy to websphere-liberty_18.0.0.3-javaee7/server.xml
diff --git a/websphere-liberty_latest/Dockerfile b/websphere-liberty_18.0.0.3-javaee8/Dockerfile
similarity index 100%
copy from websphere-liberty_latest/Dockerfile
copy to websphere-liberty_18.0.0.3-javaee8/Dockerfile
diff --git a/websphere-liberty_latest/server.xml b/websphere-liberty_18.0.0.3-javaee8/server.xml
similarity index 100%
copy from websphere-liberty_latest/server.xml
copy to websphere-liberty_18.0.0.3-javaee8/server.xml
diff --git a/websphere-liberty_kernel/Dockerfile b/websphere-liberty_18.0.0.3-kernel/Dockerfile
similarity index 100%
copy from websphere-liberty_kernel/Dockerfile
copy to websphere-liberty_18.0.0.3-kernel/Dockerfile
diff --git a/websphere-liberty_kernel/docker-server b/websphere-liberty_18.0.0.3-kernel/docker-server
similarity index 100%
copy from websphere-liberty_kernel/docker-server
copy to websphere-liberty_18.0.0.3-kernel/docker-server
diff --git a/websphere-liberty_kernel/en.html b/websphere-liberty_18.0.0.3-kernel/en.html
similarity index 100%
copy from websphere-liberty_kernel/en.html
copy to websphere-liberty_18.0.0.3-kernel/en.html
diff --git a/websphere-liberty_kernel/non_ibm_license.html b/websphere-liberty_18.0.0.3-kernel/non_ibm_license.html
similarity index 100%
copy from websphere-liberty_kernel/non_ibm_license.html
copy to websphere-liberty_18.0.0.3-kernel/non_ibm_license.html
diff --git a/websphere-liberty_kernel/notices.html b/websphere-liberty_18.0.0.3-kernel/notices.html
similarity index 100%
copy from websphere-liberty_kernel/notices.html
copy to websphere-liberty_18.0.0.3-kernel/notices.html
diff --git a/websphere-liberty_microProfile1/Dockerfile b/websphere-liberty_18.0.0.3-microProfile1/Dockerfile
similarity index 100%
copy from websphere-liberty_microProfile1/Dockerfile
copy to websphere-liberty_18.0.0.3-microProfile1/Dockerfile
diff --git a/websphere-liberty_microProfile1/server.xml b/websphere-liberty_18.0.0.3-microProfile1/server.xml
similarity index 100%
copy from websphere-liberty_microProfile1/server.xml
copy to websphere-liberty_18.0.0.3-microProfile1/server.xml
diff --git a/websphere-liberty_microProfile/Dockerfile b/websphere-liberty_18.0.0.3-microProfile2/Dockerfile
similarity index 100%
copy from websphere-liberty_microProfile/Dockerfile
copy to websphere-liberty_18.0.0.3-microProfile2/Dockerfile
diff --git a/websphere-liberty_microProfile/server.xml b/websphere-liberty_18.0.0.3-microProfile2/server.xml
similarity index 100%
rename from websphere-liberty_microProfile/server.xml
rename to websphere-liberty_18.0.0.3-microProfile2/server.xml
diff --git a/websphere-liberty_springBoot1/Dockerfile b/websphere-liberty_18.0.0.3-springBoot1/Dockerfile
similarity index 100%
copy from websphere-liberty_springBoot1/Dockerfile
copy to websphere-liberty_18.0.0.3-springBoot1/Dockerfile
diff --git a/websphere-liberty_springBoot1/server.xml b/websphere-liberty_18.0.0.3-springBoot1/server.xml
similarity index 100%
copy from websphere-liberty_springBoot1/server.xml
copy to websphere-liberty_18.0.0.3-springBoot1/server.xml
diff --git a/websphere-liberty_springBoot2/Dockerfile b/websphere-liberty_18.0.0.3-springBoot2/Dockerfile
similarity index 100%
copy from websphere-liberty_springBoot2/Dockerfile
copy to websphere-liberty_18.0.0.3-springBoot2/Dockerfile
diff --git a/websphere-liberty_springBoot2/server.xml b/websphere-liberty_18.0.0.3-springBoot2/server.xml
similarity index 100%
copy from websphere-liberty_springBoot2/server.xml
copy to websphere-liberty_18.0.0.3-springBoot2/server.xml
diff --git a/websphere-liberty_webProfile7/Dockerfile b/websphere-liberty_18.0.0.3-webProfile7/Dockerfile
similarity index 100%
copy from websphere-liberty_webProfile7/Dockerfile
copy to websphere-liberty_18.0.0.3-webProfile7/Dockerfile
diff --git a/websphere-liberty_webProfile7/server.xml b/websphere-liberty_18.0.0.3-webProfile7/server.xml
similarity index 100%
copy from websphere-liberty_webProfile7/server.xml
copy to websphere-liberty_18.0.0.3-webProfile7/server.xml
diff --git a/websphere-liberty_webProfile8/Dockerfile b/websphere-liberty_18.0.0.3-webProfile8/Dockerfile
similarity index 100%
copy from websphere-liberty_webProfile8/Dockerfile
copy to websphere-liberty_18.0.0.3-webProfile8/Dockerfile
diff --git a/websphere-liberty_webProfile8/server.xml b/websphere-liberty_18.0.0.3-webProfile8/server.xml
similarity index 100%
copy from websphere-liberty_webProfile8/server.xml
copy to websphere-liberty_18.0.0.3-webProfile8/server.xml
diff --git a/websphere-liberty_kernel/Dockerfile b/websphere-liberty_kernel/Dockerfile
index f3d9cad..cde8164 100644
--- a/websphere-liberty_kernel/Dockerfile
+++ b/websphere-liberty_kernel/Dockerfile
@@ -25,7 +25,7 @@ RUN apt-get update \
 COPY *.html /licenses/
 
 # Install WebSphere Liberty
-ENV LIBERTY_VERSION 18.0.0_03
+ENV LIBERTY_VERSION 18.0.0_04
 ARG LIBERTY_URL
 ARG DOWNLOAD_OPTIONS=""
 RUN LIBERTY_URL=${LIBERTY_URL:-$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml  | grep $LIBERTY_VERSION -A 6 | sed -n 's/\s*kernel:\s//p' | tr -d '\r' )}  \
@@ -39,7 +39,7 @@ ENV PATH=/opt/ibm/wlp/bin:$PATH
 # Add labels for consumption by IBM Product Insights
 LABEL "ProductID"="fbf6a96d49214c0abc6a3bc5da6e48cd" \
       "ProductName"="WebSphere Application Server Liberty" \
-      "ProductVersion"="18.0.0.3"
+      "ProductVersion"="18.0.0.4"
 
 # Set Path Shortcuts
 ENV LOG_DIR=/logs \
diff --git a/websphere-liberty_kernel/en.html b/websphere-liberty_kernel/en.html
index 010f812..0bc786e 100644
--- a/websphere-liberty_kernel/en.html
+++ b/websphere-liberty_kernel/en.html
@@ -14,7 +14,7 @@ the Program, the International License Agreement for Non-
 Warranted Programs (Z125-5589-05) applies.<br>
 <br>
 Program Name (Program Number):<br>
-IBM WebSphere Application Server Liberty 18.0.0.3 (5724-J08)<br>
+IBM WebSphere Application Server Liberty 18.0.0.4 (5724-J08)<br>
 <br>
 The following standard terms apply to Licensee's use of the 
 Program.<br>
@@ -157,9 +157,9 @@ change, and distribute the software to anyone and for any purpose. <br>
 An "Install" is an installed copy of the Program on a 
 physical or virtual disk made available to be executed on a computer.<br>
 <br>
-L/N:  L-ARON-B4RTM7<br>
-D/N:  L-ARON-B4RTM7<br>
-P/N:  L-ARON-B4RTM7<br>
+L/N:  L-CTUR-B4WNHE<br>
+D/N:  L-CTUR-B4WNHE<br>
+P/N:  L-CTUR-B4WNHE<br>
 <br>
 <br>
 International License Agreement for Non-Warranted Programs<br>
diff --git a/websphere-liberty_kernel/non_ibm_license.html b/websphere-liberty_kernel/non_ibm_license.html
index 8f503cc..ad017da 100644
--- a/websphere-liberty_kernel/non_ibm_license.html
+++ b/websphere-liberty_kernel/non_ibm_license.html
@@ -6,7 +6,7 @@
 <BODY>
 TERMS AND CONDITIONS FOR SEPARATELY LICENSED CODE<br>
 <br>
-IBM WebSphere Application Server Liberty 18.0.0.3<br>
+IBM WebSphere Application Server Liberty 18.0.0.4<br>
 <br>
 The IBM license agreement and any applicable information on 
 the web<br>
@@ -40,18 +40,17 @@ The following are Separately Licensed Code:<br>
 Ubuntu<br>
 <br>
 <br>
-========================================================================================================<br>
+==================================================================================<br>
 Ubuntu Operating System<br>
---------------------------------------------------------------------------------------------------------------------------------------<br>
-When IBM WebSphere Application Server Liberty is provided 
-in a container format, license information for Ubuntu packages 
-may be found in /usr/share/doc/${package}/copyright<br>
---------------------------------------------------------------------------------------------------------------------------------------<br>
-========================================================================================================<br>
-<br>
-<br>
-L/N:  L-ARON-B4RTM7<br>
-D/N:  L-ARON-B4RTM7<br>
-P/N:  L-ARON-B4RTM7<br>
+-------------------------------------------------------------------------------------------------------------------------------<br>
+When WebSphere Application Server Liberty  is provided in a 
+container format, license information for Ubuntu packages may be 
+found in /usr/share/doc/${package}/copyright<br>
+-------------------------------------------------------------------------------------------------------------------------------<br>
+==================================================================================<br>
+<br>
+L/N:  L-CTUR-B4WNHE<br>
+D/N:  L-CTUR-B4WNHE<br>
+P/N:  L-CTUR-B4WNHE<br>
 ﻿</BODY>
 </HTML>
\ No newline at end of file
diff --git a/websphere-liberty_kernel/notices.html b/websphere-liberty_kernel/notices.html
index 1f6390c..b600903 100644
--- a/websphere-liberty_kernel/notices.html
+++ b/websphere-liberty_kernel/notices.html
@@ -6,7 +6,7 @@
 <BODY>
 NOTICES AND INFORMATION<br>
 <br>
-IBM WebSphere Application Server Liberty 18.0.0.3<br>
+IBM WebSphere Application Server Liberty 18.0.0.4<br>
 <br>
 The IBM license agreement and any applicable information on 
 the web<br>
@@ -179,6 +179,7 @@ OpenWS<br>
 OSGi Materials <br>
 OSGi Core<br>
 OSGi Compendium<br>
+rxJava<br>
 Spatial4j<br>
 Swagger Annotations<br>
 Swagger Models<br>
@@ -628,8 +629,9 @@ OSGi Framework, OSGi Materials, pixman, XHTML DTDs, MXJ4,  <br>
 AspectJ Runtime,  AsjectJ Tools, IBM Eclipse SDK, IBM 
 Eclipse Help System, 3.4.3 components,  <br>
 Higgens Token Server, TPTP , EMF, Eclipse Equinox (OSGi 
-Services), Eclipselink, Eclipse JDT Core Batch Compiler, PAHO Client, 
-ECJ, Yasson, Open Liberty<br>
+Services), Eclipselink, EclipseLink ANTLR, EclipseLink Hermes JPQL 
+Parser,   Eclipse JDT Core Batch Compiler, PAHO Client, ECJ, Yasson, 
+Open Liberty<br>
 <br>
 (herein collectively referred to as "Eclipse Code"). 
 Notices and important information, including instructions for 
@@ -1387,8 +1389,10 @@ Java Server Pages (JSP) API<br>
 Java Server Pages Standard Tag Library (JSTL) <br>
 Java Transaction API (JTA) <br>
 Java Transaction API (JTA) Javadoc<br>
+javax.xml.soap-api<br>
 JBoss Jdeparser<br>
 JSON <br>
+jsr181-api<br>
 Policy<br>
 StreamBuffer<br>
 <br>
@@ -1722,6 +1726,10 @@ SVG 4 Everybody: The Program includes SVG 4 Everybody which
 IBM obtained under the terms and conditions of the following 
 license.<br>
 <br>
+Reactive-streams and Reactive-streams-tck,  which IBM 
+obtained under the terms and <br>
+conditions of the following license.<br>
+<br>
 # CC0 1.0 Universal License<br>
 <br>
 Public Domain Dedication<br>
@@ -1751,15 +1759,16 @@ This is a [human-readable summary of the Legal Code](https:
 //creativecommons.org/publicdomain/zero/1.0/) ([read the full text](https:
 //creativecommons.org/publicdomain/zero/1.0/legalcode)).<br>
 <br>
-END OF SVG 4 EVERYBODY NOTICES AND INFORMATION<br>
+END OF SVG 4 EVERYBODY, Reactive-streams, and Reactive-
+streams-tck NOTICES AND INFORMATION<br>
 ==============================================================================<br>
 <br>
 <br>
 END OF NOTICES AND INFORMATION<br>
 ==============================================================================<br>
 <br>
-L/N:  L-ARON-B4RTM7<br>
-D/N:  L-ARON-B4RTM7<br>
-P/N:  L-ARON-B4RTM7<br>
+L/N:  L-CTUR-B4WNHE<br>
+D/N:  L-CTUR-B4WNHE<br>
+P/N:  L-CTUR-B4WNHE<br>
 ﻿</BODY>
 </HTML>
\ No newline at end of file
diff --git a/websphere-liberty_latest/Dockerfile b/websphere-liberty_latest/Dockerfile
index d139763..77b64b6 100644
--- a/websphere-liberty_latest/Dockerfile
+++ b/websphere-liberty_latest/Dockerfile
@@ -22,7 +22,7 @@ RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
     appSecurity-2.0 ldapRegistry-3.0 \
     localConnector-1.0 monitor-1.0 requestTiming-1.0 restConnector-2.0 sessionCache-1.0 \
     sessionDatabase-1.0 ssl-1.0 transportSecurity-1.0 webCache-1.0 webProfile-8.0 \
-    appSecurityClient-1.0 javaee-8.0 javaeeClient-8.0 openidConnectClient-1.0 monitor-1.0 microProfile-2.0 \
+    appSecurityClient-1.0 javaee-8.0 javaeeClient-8.0 openidConnectClient-1.0 monitor-1.0 microProfile-2.1 \
   && if [ ! -z $REPOSITORIES_PROPERTIES ]; then rm /opt/ibm/wlp/etc/repositories.properties; fi \
   && rm -rf /output/workarea /output/logs
 
diff --git a/websphere-liberty_microProfile/Dockerfile b/websphere-liberty_microProfile2/Dockerfile
similarity index 99%
rename from websphere-liberty_microProfile/Dockerfile
rename to websphere-liberty_microProfile2/Dockerfile
index 74e08c4..91f6c1e 100644
--- a/websphere-liberty_microProfile/Dockerfile
+++ b/websphere-liberty_microProfile2/Dockerfile
@@ -19,7 +19,7 @@ ARG REPOSITORIES_PROPERTIES=""
 RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
   && echo $REPOSITORIES_PROPERTIES > /opt/ibm/wlp/etc/repositories.properties; fi \
   && installUtility install --acceptLicense \
-    appSecurity-2.0 ldapRegistry-3.0 localConnector-1.0 microProfile-2.0 \
+    appSecurity-2.0 ldapRegistry-3.0 localConnector-1.0 microProfile-2.1 \
     monitor-1.0 requestTiming-1.0 restConnector-2.0 sessionCache-1.0 \
     sessionDatabase-1.0 ssl-1.0 transportSecurity-1.0 webCache-1.0 \
     webProfile-8.0 \
diff --git a/websphere-liberty_microProfile1/server.xml b/websphere-liberty_microProfile2/server.xml
similarity index 92%
copy from websphere-liberty_microProfile1/server.xml
copy to websphere-liberty_microProfile2/server.xml
index 58c30b9..5272d6b 100644
--- a/websphere-liberty_microProfile1/server.xml
+++ b/websphere-liberty_microProfile2/server.xml
@@ -3,7 +3,7 @@
 
     <!-- Enable features -->
     <featureManager>
-        <feature>microProfile-1.4</feature>
+        <feature>microProfile-2.1</feature>
         <feature>monitor-1.0</feature>
     </featureManager>
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #5185; 46419209; `amd64` (`websphere-liberty`):

```console

$ bashbrew build websphere-liberty:beta
Using bashbrew/cache:b1e5a922b0c0c763cc3d1b696127142d48950b1db9e7823f1748a90076ca8d51 (websphere-liberty:beta)
Tagging websphere-liberty:beta

$ test/run.sh websphere-liberty:beta
testing websphere-liberty:beta
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-kernel
Using bashbrew/cache:d9366c5b3e9582752be7e7c676fe36c0574152df0dfd2003d66c1ce9bb806cde (websphere-liberty:18.0.0.4-kernel)
Tagging websphere-liberty:18.0.0.4-kernel
Tagging websphere-liberty:kernel

$ test/run.sh websphere-liberty:18.0.0.4-kernel
testing websphere-liberty:18.0.0.4-kernel
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-springBoot1
Using bashbrew/cache:e230aba11b432e1258116b11b83312186cd7ff9d53d7d78a2a5d327a40db0c11 (websphere-liberty:18.0.0.4-springBoot1)
Tagging websphere-liberty:18.0.0.4-springBoot1
Tagging websphere-liberty:springBoot1

$ test/run.sh websphere-liberty:18.0.0.4-springBoot1
testing websphere-liberty:18.0.0.4-springBoot1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-webProfile7
Using bashbrew/cache:3de2f8008c030a2eeee7f56f650f98871055d4f7f7079fb4a0637f03fe18badf (websphere-liberty:18.0.0.4-webProfile7)
Tagging websphere-liberty:18.0.0.4-webProfile7
Tagging websphere-liberty:webProfile7

$ test/run.sh websphere-liberty:18.0.0.4-webProfile7
testing websphere-liberty:18.0.0.4-webProfile7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-javaee7
Using bashbrew/cache:fd37ae256fd9913516d9eaee2f19d662e68f40e083600475cd10d790deca7601 (websphere-liberty:18.0.0.4-javaee7)
Tagging websphere-liberty:18.0.0.4-javaee7
Tagging websphere-liberty:javaee7

$ test/run.sh websphere-liberty:18.0.0.4-javaee7
testing websphere-liberty:18.0.0.4-javaee7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-javaee8
Using bashbrew/cache:a20c274a4eb025a22ffb6106d2ff17961a6214134cee367451222556d9dce237 (websphere-liberty:18.0.0.3-javaee8)
Tagging websphere-liberty:18.0.0.3-javaee8

$ test/run.sh websphere-liberty:18.0.0.3-javaee8
testing websphere-liberty:18.0.0.3-javaee8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-webProfile8
Using bashbrew/cache:680f60ec9022d26f1e6c395522ba90df331e12845bc20c8610a245a0bb9807bd (websphere-liberty:18.0.0.3-webProfile8)
Tagging websphere-liberty:18.0.0.3-webProfile8

$ test/run.sh websphere-liberty:18.0.0.3-webProfile8
testing websphere-liberty:18.0.0.3-webProfile8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-microProfile1
Using bashbrew/cache:62dde46b779b94e9e7f217d4435e68a92b15cc0f2871c5499f5740a1d2b90a65 (websphere-liberty:18.0.0.3-microProfile1)
Tagging websphere-liberty:18.0.0.3-microProfile1

$ test/run.sh websphere-liberty:18.0.0.3-microProfile1
testing websphere-liberty:18.0.0.3-microProfile1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-microProfile2
Using bashbrew/cache:0083edefcf115a471c6b040899b73be5fd77a07b8a35e2245366b638d5eed100 (websphere-liberty:18.0.0.3-microProfile2)
Tagging websphere-liberty:18.0.0.3-microProfile2

$ test/run.sh websphere-liberty:18.0.0.3-microProfile2
testing websphere-liberty:18.0.0.3-microProfile2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-springBoot2
Using bashbrew/cache:1a6d6f84b507e9d4734b8f161f75211164a216d24aaa2842e19f3c54715523b1 (websphere-liberty:18.0.0.3-springBoot2)
Tagging websphere-liberty:18.0.0.3-springBoot2

$ test/run.sh websphere-liberty:18.0.0.3-springBoot2
testing websphere-liberty:18.0.0.3-springBoot2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-kernel
Using bashbrew/cache:0589c8a2e2531f0af800635f7e7bf27ee8b3796cc89db9dd4a472fbaf1dfc4ec (websphere-liberty:18.0.0.3-kernel)
Tagging websphere-liberty:18.0.0.3-kernel

$ test/run.sh websphere-liberty:18.0.0.3-kernel
testing websphere-liberty:18.0.0.3-kernel
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-springBoot1
Using bashbrew/cache:dae7206effd33180173c910decdec59d7d187f594512ee1da00d935bd1c6b39e (websphere-liberty:18.0.0.3-springBoot1)
Tagging websphere-liberty:18.0.0.3-springBoot1

$ test/run.sh websphere-liberty:18.0.0.3-springBoot1
testing websphere-liberty:18.0.0.3-springBoot1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-webProfile7
Using bashbrew/cache:ecaa5e82edabd51ecada1397f5e9df2fef03f8c4a7121772f77dc722c8e82bf5 (websphere-liberty:18.0.0.3-webProfile7)
Tagging websphere-liberty:18.0.0.3-webProfile7

$ test/run.sh websphere-liberty:18.0.0.3-webProfile7
testing websphere-liberty:18.0.0.3-webProfile7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.3-javaee7
Using bashbrew/cache:933c9dee215f2ab1791b43ce923a1ca7d2faabee6108907a9b457feb94774f61 (websphere-liberty:18.0.0.3-javaee7)
Tagging websphere-liberty:18.0.0.3-javaee7

$ test/run.sh websphere-liberty:18.0.0.3-javaee7
testing websphere-liberty:18.0.0.3-javaee7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-javaee8
Using bashbrew/cache:9d0547b7fd2ca1f51bb56f25256f101cb95f8ea9e4ba8a55882e4ec5e4faeede (websphere-liberty:18.0.0.4-javaee8)
Tagging websphere-liberty:18.0.0.4-javaee8
Tagging websphere-liberty:javaee8
Tagging websphere-liberty:latest

$ test/run.sh websphere-liberty:18.0.0.4-javaee8
testing websphere-liberty:18.0.0.4-javaee8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-webProfile8
Using bashbrew/cache:0bf3c88161d5b42c34b75ce74cbce3b053158ae6942d0c5bcf0dda496d4bb2de (websphere-liberty:18.0.0.4-webProfile8)
Tagging websphere-liberty:18.0.0.4-webProfile8
Tagging websphere-liberty:webProfile8

$ test/run.sh websphere-liberty:18.0.0.4-webProfile8
testing websphere-liberty:18.0.0.4-webProfile8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-microProfile1
Using bashbrew/cache:0cadabadab45f7410ffb606ad7b43823369afb22e706ae74ddb4ee8ac09f710c (websphere-liberty:18.0.0.4-microProfile1)
Tagging websphere-liberty:18.0.0.4-microProfile1
Tagging websphere-liberty:microProfile1

$ test/run.sh websphere-liberty:18.0.0.4-microProfile1
testing websphere-liberty:18.0.0.4-microProfile1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-microProfile2
Using bashbrew/cache:beb68634913c261939c0ba3d368216da4c100bdb68a3057770e06f20aa7e9327 (websphere-liberty:18.0.0.4-microProfile2)
Tagging websphere-liberty:18.0.0.4-microProfile2
Tagging websphere-liberty:microProfile2

$ test/run.sh websphere-liberty:18.0.0.4-microProfile2
testing websphere-liberty:18.0.0.4-microProfile2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build websphere-liberty:18.0.0.4-springBoot2
Using bashbrew/cache:db1af97aab668893a4248fde590e5077b9668861ed0c4bb26d17a7c48f649fd1 (websphere-liberty:18.0.0.4-springBoot2)
Tagging websphere-liberty:18.0.0.4-springBoot2
Tagging websphere-liberty:springBoot2

$ test/run.sh websphere-liberty:18.0.0.4-springBoot2
testing websphere-liberty:18.0.0.4-springBoot2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**